facturacion_mexico/facturacion_fiscal/services/ffm_reconciliation.py

```
@@ -251,6 +275,51 @@ def _reconcile_ffm(ffm_name: str) -> dict:
+		return False
+	b_si = frappe.db.get_value("Sales Invoice", {"ffm_substitution_source_uuid": uuid}, "name")
+	if not b_si:
+		return False
+	return bool(
+		frappe.db.get_value(
+			"Factura Fiscal Mexico", {"sales_invoice": b_si, "status": FiscalStates.TIMBRADO}, "name"
+		)
+	)
+
+
+def _reconcile_substitution_documental(ffm):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_reconcile_substitution_documental`

Add return type annotation: `None`

(ANN202)